addons/sourcemod/translations/csgo_weaponstickers.phrases.txt

```diff
@@ -197,4 +219,146 @@
 		"pt"		"Todos Slots"
 		"ru"		"Все слоты"
 	}
+	"Set Wear"
+	{
+		"en"		"Set Wear"
+		"chi"	    "设置印花磨损"
+		"pt"		"Definir desgaste"
+		"ru"		"Изменить качество наклеек"
+	}
+	"Stickers Wear"
+	{
+		"en"		"Stickers Wear"
+		"chi"		"印花磨损设置菜单"
+		"pt"		"Vestir adesivos"
+		"ru"		"Качество наклеек"
+	}
+	"Set Wear Title"
+	{
+		"#format"	"{1:i},{2:.1f}"
+		"en"		"Select Slot # {1} Wear \n Current Wear : {2}"
+		"chi"       "设置# {1} 位置印花的磨损 \n 当前磨损 : {2}"
+		"pt"		"Selecione Slot # {1} Vestir \n Desgaste Atual : {2}"
+		"ru"		"Выбран слот # {1} для качества наклейки \n Текущее качество : {2}"
+	}
+	"Reset Wear"
+	{
+		"en"		"Reset Wear"
+		"chi"		"重置印花磨损"
+		"pt"		"Reiniciar desgaste"
+		"ru"		"Сбросить износ"
+	}
+	"Add Wear"
+	{
+		"en"		"Add Wear"
+		"chi"		"增加印花磨损"
+		"pt"		"Adicionar desgaste"
+		"ru"		"Увеличить износ"
+	}
+	"Del Wear"
+	{
+		"en"		"Del Wear"
+		"chi"		"减少印花磨损"
+		"pt"		"Del Wear"
+		"ru"		"Уменьшить износ"
+	}
+	"Wear Updated"
+	{
+		"en"		"Wear Updated"
+		"chi"		"武器印花的磨损已更新"
+		"pt"		"Desgaste atualizado"
+		"ru"		"Износ обновлен"
+	}
+	"Set Rotation"
+	{
+		"en"		"Set Wear"
+		"chi"	    "设置印花磨损"
+		"pt"		"Definir desgaste"
+		"ru"		"Изменить поворот наклеек"
+	}
+	"Stickers Rotation"
+	{
+		"en"		"Stickers Wear"
+		"chi"		"印花磨损设置菜单"
+		"pt"		"Vestir adesivos"
+		"ru"		"Поворот наклеек"
+	}
```

Comment:
Is the translation right here? Wouldn't it be translation for rotation of stickers?